Light.json

```diff
@@ -55967,6 +55914,338 @@
+        "hyperspeed": 4,
+        "deploy": 3,
+        "forfeit": 6,
+        "icons": [
+          "Pilot",		
+          "Nav Computer",
+          "Scomp Link",
+		  "Republic",
+		  "Episode I",
+		  "Clone Army"
+        ],
+        "gametext": "May add 1 pilot or passenger. Permanent pilot provides ability of 2. Opponent may not 'react' to or from here. During battle, if about to be lost before the damage segment, lost at end of battle instead.",
+        "lore": "Blank."
+      },	
+	
+    {
+      "id": 6870,
+	  "gempId": "215_1",
+      "side": "Light",
+      "rarity": "U",
+      "set": "215",
+      "printings": [
+        {
+          "set": "215"
+        }
+      ],
+      "front": {
+        "title": "•Odd Ball's Torrent Starfighter",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual15-Light/large/oddballstorrentstarfighter.gif?",
+        "type": "Starship",
+        "subType": "Starfighter: V-19 Torrent Starfighter",
+        "uniqueness": "*",
+        "destiny": 3,
+        "power": 1,
+        "maneuver": 3,
+        "hyperspeed": 3,
+        "deploy": 1,
+        "forfeit": 3,
+        "icons": [
+          "Nav Computer",
+          "Scomp Link",
+		  "Republic",
+		  "Episode I",
+		  "Clone Army"
+        ],
+        "gametext": "May add 1 clone pilot. Odd Ball deploys -1 aboard. While Odd Ball piloting, power and hyperspeed +1, and adds one destiny to total power. Immune to attrition < 3 (< 5 while Odd Ball piloting).",
+        "lore": "Blank."
+      },	
+	
+    {
+      "id": 6871,
+	  "gempId": "214_21",
+      "side": "Light",
+      "rarity": "U",
+      "set": "214",
+      "printings": [
+        {
+          "set": "214"
+        }
+      ],
+      "front": {
+        "title": "•Plo Koon's Jedi Starfighter",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual14-Light/large/plokoonsjedistarfighter.gif?",
+        "type": "Starship",
+        "subType": "Starfighter: Jedi Interceptor",
+        "uniqueness": "*",
+        "destiny": 2,
+        "power": 2,
+        "maneuver": "*",
+        "hyperspeed": 6,
+        "deploy": 2,
+        "forfeit": 5,
+        "icons": [
+          "Nav Computer",
+          "Scomp Link",
+		  "Republic",
+		  "Episode I",
+		  "Clone Army"
+        ],
+        "gametext": "May add 1 Jedi pilot. *Maneuver = pilot's ability. While Plo piloting, power +2, immune to attrition < 3, and once per turn, may cancel and redraw your weapon or battle destiny just drawn here.",
+        "lore": "Blank."
+      },	
+    {
+      "id": 6872,
+	  "gempId": "214_15",
+      "side": "Light",
+      "rarity": "U",
+      "set": "214",
+      "printings": [
+        {
+          "set": "214"
+        }
+      ],
+      "front": {
+        "title": "•Ahsoka's Shoto Lightsaber",
+        "imageUrl": "https://res.starwarsccg.org/cards/Virtual14-Light/large/ahsokasshotolightsaber.gif?",
+        "type": "Weapon",
+        "subType": "Character",
+        "destiny": 2,
```

Comment:
Sorry I missed this earlier, but destiny, power, forfeit, deploy, maneuver, landspeed, etc, etc should all be Strings now.